nxtool/log_providers/elastic.py

```diff
@@ -180,23 +201,26 @@ def commit(self):
         """Process list of dict (yes) and push them to DB """
         self.total_objs += len(self.nlist)
         count = 0
-        full_body = ""
-        items = []
+
+        events = list()
         for entry in self.nlist:
-            items.append({"index" : {
-                "_index": self.index,
-                "_type" : "events"}})
-            entry['whitelisted'] = "false"
-            entry['comments'] = "import:"+str(datetime.datetime.now())
-            # go utf-8 ?
-            items.append(entry)
+            event = Event()
+            for key, value in entry.items():
+                setattr(event, key, value)
+
+            event.whitelisted = False
+            event.comments = "import on"+str(datetime.datetime.now())
+            events.append(event)
             count += 1
-        try:
-            self.client.bulk(body=items)
-        except TransportError:
-            for meta, item in zip(items[0::2], items[1::2]): #We build tuples in items (items[0], items[1]), (items[2], items[3]) ...
-                self.client.bulk(body=[meta, item])
 
+        actions = (item for items in [[{'index': {'_index': 'nxapi', '_type': 'events'}}, d.to_dict()] for d in events] for item in items)
```

Comment:
I don't think that it's wise to use `item` and `items` two times, for different values :/